reporter/html.js

```diff
@@ -723,6 +723,20 @@ QUnit.testDone(function( details ) {
 		time.innerHTML = details.runtime + " ms";
 		testItem.insertBefore( time, assertList );
 	}
+
+	// Only add this information if stackTrace is supported
+	if ( Error().stack ) {
```

Comment:
There may be browsers that don't generate the `stack` property when calling just `Error()`, but will provide the property when catching an error using `try/catch`. We either have to take that into account and use a different check, or we have to verify that the behaviour isn't actually happening anywhere. In that case we could also simplify `sourceFromStacktrace`, removing the `try/catch`.